key-management-scenarios.md

```diff
@@ -0,0 +1,19 @@
+# Notary Key Management Scenarios
+In order to allow for end to end signing of artifacts, Notary v2 will require the use of cryptographic keys. These scenarios define how these keys will be generated, distributed, and trusted in Notary v2.
+
+## Scenarios
+
+### Scenario #1: A developer signs a package for the first time
+The first time a developer signs a package, they will need to set up a signing key and their local environment.
+  1. Generate a developer signing key.
+  1. Create a local repository that delegates to the developer key notary-tool --create-local-repository, notary-tool --delegate developer-key.pub
```

Comment:
Create a local repository is a little specific, and I'm not sure what it means in the sense of a registry. 
Is this:
- Developers provides a signing key, or reference to a signing key - where they get the key is outside the scope
- Developer signs local content

The details for the cli are design choices, not part of the scenario.
We do need to start a UX discussion, which will get shaped shortly as part of a prototyping phase.